vaibhav_kaushik.md

```diff
@@ -0,0 +1,4 @@
+Name: Vaibhav Kaushik
```

Comment:
The content is in markdown code format, like 

```This is the contetn nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnttt``` , see the work of priyanka 's PR.

Reply:
I have made the required changes, to make the text in markdown code format.